lambdas/handlers/create_document_reference_handler.py

```diff
@@ -19,23 +20,42 @@
 def lambda_handler(event, context):
     logger.info("Starting document reference creation process")
 
+    try:
+        document_type = (event["queryStringParameters"]["documentType"]).upper()
+        if document_type in SupportedDocumentTypes.list_names():
+            logger.info("Provided document is supported")
+        else:
+            raise Exception("Provided document type is not supported")   
```

Comment:
In Java, it's good practice to never raise / throw base Exception class. Not sure about Python, but this gives me the willies.